### PR TITLE
Made a MOYO navigation

### PR DESCRIPTION
**Pull request by ia-lerimund**
Again in the tests, a millimeter error.
The bottom margin for the picture was made to remove this problem, it partially helped, but now I don’t see what the difference is.

- [DEMO LINK](https://ia-lerimund.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ia-lerimund.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
fix this
Screenshot 2022-10-26 at 10 51 42

File: src/style.css [hunk 1]
```diff
@@ -1 +1,77 @@
...
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Serif:opsz,wght@8..144,400;8..144,700&family=Roboto:wght@500&display=swap");
+
+html,
+body,
+a,
+img,
+ul,
+li {
+  margin: 0;
+  box-sizing: border-box;
```
Comment (stanislav-topikha) [inline src/style.css:11]: set box-sizing to *

File: src/style.css [hunk 2]
```diff
@@ -1 +1,77 @@
...
+}
+
+body {
+  font-family: "Roboto", sans-serif;
```
Comment (stanislav-topikha) [inline src/style.css:15]: move it to `html`, add fallback fonts

File: src/style.css [hunk 3]
```diff
@@ -1 +1,77 @@
...
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 36.9px;
```
Comment (stanislav-topikha) [inline src/style.css:72]:
```suggestion
  right: 0;
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,77 @@
...
+  text-transform: uppercase;
+  color: black;
+  padding: 22.5px 0;
+  white-space: nowrap;
+  margin-right: 20px;
+  cursor: pointer;
+}
+
+.link:hover {
+  color: #00acdc;
```
Comment (stanislav-topikha) [inline src/style.css:56]: move same  colors to the var

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 5]
```diff
@@ -1 +1,87 @@
...
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+body {
+  font-size: 12px;
+}
```
Comment (ivanchuk-yurii) [inline src/style.css:27]: it's better to move this into separate file

File: src/style.css [hunk 6]
```diff
@@ -1 +1,87 @@
...
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  right: 0;
+  left: 0;
+  bottom: -0.6px;
```
Comment (ivanchuk-yurii) [inline src/style.css:84]: why not `0`?

File: src/index.html [hunk 7]
```diff
@@ -8,7 +8,49 @@
...
+      <a href="#home">
+        <img 
+          class="header-logo" 
+          src="./images/logo.png" 
+          alt="Moyo logo"
+        > 
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-links">
```
Comment (ivanchuk-yurii) [inline src/index.html:22]: why do you use plural in class name? it's better to name `nav-item`

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 6 above)
Comment (ia-lerimund) [reply src/style.css:84]: Before that, it did not want to pass tests without specifying this parameter as -0.6, but now it has passed with 0. Thank you.

File: src/style.css [hunk 8]
```diff
@@ -1 +1,61 @@
...
+  height: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.link {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  padding: 22.5px 0;
```
Comment (SanyaBratashchuk) [inline src/style.css:33]:
```suggestion
  padding: 23px 0;
```

File: src/style.css [hunk 9]
```diff
@@ -1 +1,61 @@
...
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  right: 0;
+  left: 0;
+  bottom: -0.6px;
```
Comment (SanyaBratashchuk) [inline src/style.css:58]:
```suggestion
  bottom: 0;
```
Try with padding 23 instead of 22.5

**Review by SanyaBratashchuk — APPROVED** [review]

File: src/style.css (hunk 9 above)
Comment (ia-lerimund) [reply src/style.css]: Thank you very much, it worked :)

**Review by stanislav-topikha — APPROVED** [review]

**Pull request closed without merging**